common/src/test/java/org/apache/sedona/common/raster/FunctionsTest.java

```diff
@@ -67,6 +67,15 @@ public void valueWithMultibandRaster() throws TransformException {
         assertEquals(255d, PixelFunctions.value(multiBandRaster, point(4.5d,4.5d), 4), 0.1d);
     }
 
+    @Test
+    public void testPixelAsPoint() throws FactoryException, TransformException {
+        GridCoverage2D emptyRaster = RasterConstructors.makeEmptyRaster(1, 5, 10, 123, -230, 8);
+        Geometry actualPoint = PixelFunctions.getPixelAsPoint(emptyRaster, 0, 0);
```

Comment:
Please add a few more test cases that test different pixel locations.

Reply:
Added more tests